luminoso_api/upload.py

```diff
@@ -98,12 +102,17 @@ def main():
              "(overrides -a)",
         action="store_true")
     parser.add_argument('-r', '--readers', metavar='LANG=READER',
-        help="Custom reader to use, in a form such as 'ja=mecab.ja,"
+        help="Custom reader to use, in a form such as 'ja=mecab.ja, "
              "en=freeling.en'")
     parser.add_argument('-u', '--username', default=None,
         help="username (defaults to your username on your computer)")
     parser.add_argument('-p', '--password', default=None,
         help="password (you can leave this out and type it in later)")
+    parser.add_argument('-d', '--date-format', default='iso',
+        help="format string for parsing dates, following http://strftime.org/"
+             "\nDefault is 'iso', which is '%Y-%m-%dT%H:%M:%S+00:00'\n"
+             "Can also pass 'epoch' for epoch time or 'us-standard' for"
+             "'%m/%d/%y'")
```

Comment:
I tried to actually run `lumi-upload --help` in the branch, and it crashed while trying to format this line (!).  Apparently if you want literal percent signs in your help strings, you have to use `%%` (see https://docs.python.org/3.4/library/argparse.html#help).

Also I am pretty sure that when actually put together you wil find a missing space between lines 114 and 115.